### ISSUE TITLE
OpenGraph should show profile picture as preview image when sharing a link to a profile

### ISSUE THREAD

**Issue by trwnh** [tootsuite/mastodon#6606]
During some Twitter spring cleaning, I linked to my mastodon profile, but the link preview looks bland: https://i.imgur.com/R0DlaSz.png

Links to toots with media attached show the attachment as the OG image; why not also show the profile picture when there is no media attached to toots, or when linking to the profile directly?

* * * *

- [x] I searched or browsed the repo’s other issues to ensure this is not a duplicate.
- [x] This bug happens on a [tagged release](https://github.com/tootsuite/mastodon/releases) and not on `master` (If you're a user, don't worry about this).

**Comment by northerner** [comment]
I've checked this with Twitter's OG tag validator (https://cards-dev.twitter.com/validator) using your profile URL (https://mastodon.social/@trwnh) and it appears to show the profile image as expected.

The profile page uses the avatar image as expected in OG tag: https://github.com/tootsuite/mastodon/blob/master/app/views/accounts/_og.html.haml

However, it does appear to be the case that when linking to to stream entries it will only add the avatar image if the entry is not of type "Status" https://github.com/tootsuite/mastodon/blob/master/app/views/stream_entries/_og_image.html.haml#L32

Would the OG tags line 33-35 in the partial linked above copied to the else block starting line 30 solve this issue?

**Comment by Gargron** [comment]
>However, it does appear to be the case that when linking to to stream entries it will only add the avatar image if the entry ~~~is not of type "Status"~~~ **does not have any media attachments**

**Comment by Gargron** [comment]
TL;DR we do everything correctly, no clue why Twitter didn't show you the image, it's up to them.

**Comment by trwnh** [comment]
I'm rather confused as well -- it seems to have loaded at some point within the past 7 hours, but not immediately. Strange. https://i.imgur.com/5CtbgqM.png
